demisto_sdk/commands/common/git_util.py

```diff
@@ -325,7 +325,7 @@ def modified_files(
             # if remote does not exist we are checking against the commit sha1
             else:
                 committed = {
-                    Path(os.path.join(item.a_path))
+                    Path(os.path.join(item.a_path))  # type: ignore
```

Comment:
do you get a mypy failure without it? can we add type hints instead?

Reply:
Its a library module... which has typing str|None... this is the reason for the mypy failure